mutiny-core/src/keymanager.rs

```diff
@@ -156,21 +157,21 @@ impl<S: MutinyStorage> SignerProvider for PhantomKeysManager<S> {
         self.inner.read_chan_signer(reader)
     }
 
-    fn get_destination_script(&self) -> Script {
+    fn get_destination_script(&self) -> Result<Script, ()> {
         let mut wallet = self.wallet.wallet.try_write().unwrap();
```

Comment:
should map this as an err

Reply:
This is a trait we are inheriting and the signature they're specifying.